.github/ISSUE_TEMPLATE/Pull_request.md

```diff
@@ -0,0 +1,19 @@
+ **What this PR does**
+<< Please describe briefly what this PR is trying to solve >>
+
+**Changes you made**
+<< List all the changes you made >>
+
+**Screenshot/ CodeSnippet**
+<< Attach relevant screenshots or code snippets related to this PR >>
+
+**Test Configuration**:
+<< Please describe the tests that you ran to verify your changes >>
+
+**Follow up**
+<< Anything you want to share >>
+
+
+PS: Please squash your commits before submitting PR
```

Comment:
Not necessary. You can remove this line. We could `Squash and Merge` if required.
Everything else okay 👍

Reply:
@umeshp7 Done!